x/emissions/types/validations.go

```diff
@@ -663,52 +663,61 @@ func (msg *RemoveRegistrationRequest) Validate() error {
 }
 
 // helper for stake validation functions
-func stakeValidateHelper(addr []string, amount cosmosMath.Int) error {
+func stakeValidateHelper(addr []string, amount cosmosMath.Int, allowZeroAmount bool) error {
```

Comment:
Better comment here would be useful as this helper is used several times below.  Might supersede my suggestion to comment each function below if it's written well